packages/kbn-es/src/serverless_resources/project_roles/es/roles.yml

```diff
@@ -50,6 +53,8 @@ admin:
 
 # temporarily added for testing purpose
 system_indices_superuser:
+  metadata:
+    _reserved: true
```

Comment:
@slobodanadamovic This seems to be the reason ES was having a hard time. I removed the `_public` metadata and it worked locally. We'll see what CI thinks.